TempleEight/docs/getting-started.md

```diff
@@ -9,7 +9,7 @@ Welcome to the complete Temple getting started guide!
 
 This tutorial will walk you through developing your own applications with Temple, from start to finish.
 
-We're going to assume you've already installed the Temple CLI, as per the [Installation Guide](../installation). 
+We're going to assume you've already installed the Temple CLI, as per the [Installation](installation) guide. 
```

Comment:
Think I prefer this:

```suggestion
We're going to assume you've already installed the Temple CLI, as per the [Installation guide](installation. 
```

Reply:
Reasoning was that then it seems funny which they're not in the `guide` category, perhaps I should rename the `guide` category, or perhaps I should have just left them in `The Basics` category. Smh it's been a long day